experimental/dds/tree2/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -195,7 +305,137 @@ export function applyMoveEffectsToMark<T>(
 	consumeEffect: boolean,
 	composeChildren?: (a: T | undefined, b: T | undefined) => T | undefined,
 ): Mark<T>[] {
-	if (isMoveMark(mark)) {
+	if (isTransientEffect(mark)) {
+		if (isMoveDestination(mark.attach)) {
+			if (isMoveSource(mark.detach)) {
+				return [mark];
```

Comment:
I'm guessing we're not doing anything because move effects are only ever meant to impact the end points of moves?
A comment would be nice.